outlines/ui-ux-outline.md

```diff
@@ -0,0 +1,32 @@
+# UI / UX
+
+1. Building reusable components to encapsulate UX patterns
+  1. Pure vs impure in general
```

Comment:
Is this about recommending that people minimize the number of "impure" components? Also, we should come up with another name, perhaps "smart components"? impure sounds a bit weird for me.